<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Vino" failures="2" tests="2" name="t1t2 and listener and externaldp(failed)(failed)" time="0.329" errors="0" timestamp="2020-05-08T09:11:16 IST">
  <testcase classname="Testlearn.t1t2" name="Testcase1" time="0.037">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Testlearn.t1t2.Testcase1(t1t2.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Testcase1 -->
  <testcase classname="Testlearn.t1t2" name="Testcase1" time="0.003">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Testlearn.t1t2.Testcase1(t1t2.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Testcase1 -->
  <testcase classname="Testlearn.t1t2" name="Testcase2" time="0.0">
    <ignored/>
  </testcase> <!-- Testcase2 -->
</testsuite> <!-- t1t2 and listener and externaldp(failed)(failed) -->
